README.md

```diff
@@ -23,6 +24,24 @@ Read more about build-info and build integration in Artifactory [here](https://w
 
 ## Using build-info-go as a CLI
 
+### Clone and Build the CLI
+
+Clone the build-info-go project by executing the following command:
+```
+git clone https://github.com/jfrog/build-info-go
+```
+Build the project by navigating to the project folder and executing the following commands.
```

Comment:
Build the project by navigating to the project folder and executing the following commands.
-->
CD to the root directory of the cloned project and build the sources as follows.